AndroidBound/src/main/java/solutions/alterego/androidbound/binding/types/CommandBinding.java

```diff
@@ -76,7 +88,11 @@ private void retryExecute(View parameter, Object parameter2) {
                     try {
                         mInfo.invoke(getSubject(), parameter2);
                     } catch (Exception e) {
-                        execute(parameter);
+                        if (mDebugMode) {
```

Comment:
Why would you change this behaviour?

Reply:
Oh, you're right, good catch - it should go to a single parameter and try that one as the last resort. Removing it.